src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -257,6 +257,28 @@ Capabilities with `CT≠0` are sealed against modification and cannot be derefer
 Instructions that operate on capabilities will produce a result with a cleared {ctag} if the source capability is sealed and the operation would alter its address, bounds, or permissions.
 Extensions that augment capability metadata must describe their interaction(s) with sealed capabilities.
 
+#Begin new since last ARC review#
+
+Given a capability with `CT=0`,
+deriving a capability with `CT≠0` is termed "sealing"
+(or "sealing with type ``x``" when a particular output ``CT=x`` is meant).
+In the other direction,
+deriving a `CT=0` capability from a `CT≠0` capability is termed "unsealing"
+(or "unsealing from type ``x``" when a particular input ``CT=x`` is meant).
+In general, each of these actions may require *authority*
+to operate at the non-zero type;
+extensions will specify how software expresses this authority
+for types not defined above.
+
+[#sec_cap_type_ambient]
+Ambient sealing type::
+Some capability types are said to be "ambiently available" (or just "ambient")
+if they do not require specific authority to seal a capability (with that type).
+Type 1 is considered ambiently available on a given platform
+if <<section_rvy_sentry_insn_ext>> is available on that platform.
```

Comment:
I still strongly believe this should be a mandatory feature in the base ISA that CHERIoT's software platform can choose to not take advantage of and that CHERIoT's extra jump-and-assert-for-CFI instructions can choose to forbid as inputs (and perhaps forbid for the baseline RISC-V ret too). But that's not for this PR...

Reply:
A consequence of this PR's refactoring is that all non-zero otypes are entirely a function of the encoding in use rather than needing something more akin to a central registry of otypes/concepts in the base spec that may or may not be supported by a particular platform.